.github/workflows/netlify_cms_config_validation.yml

```diff
@@ -0,0 +1,35 @@
+name: Netlify CMS config validation
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v1
+    - name: Parse config file
+      id: config
+      run: |
+        sudo snap install yq
+        echo "##[set-output name=result;]$(yq r ${{ env.CONFIG_PATH }} --tojson)"
+    - name: Validate config
+      uses: actions/github-script@v3
```

Comment:
The [`actions/github-script`](https://github.com/actions/github-script) GitHub Action allows us to write JavaScript-like code but with easy access to the GitHub API and the workflow run context.